ui-v2/app/routes/dc/services/show.js

```diff
@@ -19,6 +19,7 @@ export default Route.extend({
       return {
         ...model,
         ...{
+          // Nodes happen to be the ServiceInstances here
           items: model.item.Nodes,
         },
```

Comment:
A note here to the old 'noun'

Reply:
Are they still `Nodes` in the API?

Reply:
Yeah the API is still the same as always. The API here talks about `Nodes`, what we actually mean here from a user perspective is `ServiceInstances`.